akka-bench-jmh/src/main/scala/akka/actor/ForkJoinActorBenchmark.scala

```diff
@@ -72,6 +72,56 @@ class ForkJoinActorBenchmark {
 
   @Benchmark
   @Measurement(timeUnit = TimeUnit.MILLISECONDS)
+  @OperationsPerInvocation(totalMessagesLessThanCores)
+  def pingPongLessActorsThanCores(): Unit = {
+    val pingPongs =
+      for {
+        i <- 1 to lessThanCoresActorPairs
+      } yield {
+        val ping = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
+        val pong = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
+        (ping, pong)
+      }
+
+    pingPongs.foreach { case (ping, pong) => ping.tell(message, pong) }
+
+    pingPongs.foreach {
+      case (ping, pong) =>
+        val p = TestProbe()
+        p.watch(ping)
+        p.expectTerminated(ping, timeout)
+        p.watch(pong)
+        p.expectTerminated(pong, timeout)
+    }
+  }
+
+  @Benchmark
+  @Measurement(timeUnit = TimeUnit.MILLISECONDS)
+  @OperationsPerInvocation(totalMessagesMoreThanCores)
+  def pingPongMoreActorsThanCores(): Unit = {
+    val pingPongs =
+      for {
+        i <- 1 to moreThanCoresActorPairs
+      } yield {
+        val ping = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
+        val pong = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
+        (ping, pong)
+      }
+
+    pingPongs.foreach { case (ping, pong) => ping.tell(message, pong) }
```

Comment:
There must be enough messages in flight so that the mailbox aways has some messages to iterate over for the `throughput` setting to be effective. I think it would be enough to send as many messages here initially as the `throughput` setting (or make it x2).